File: HIMS/DTO/Appointment/CreateAppointmentDto.cs
using HIMS.DTO.Patient;

namespace HIMS.DTO.Appointment
{
    public class CreateAppointmentDto:PatientCheckDto
    {
        public Guid DoctorId { get; set; }
        public DateTime AppointmentDate { get; set; }
        public string BloodGroup { get; set; }
        public string Gender { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public DateOnly DateOfBirth { get; set; }
        public string ReasonForVisit { get; set; }
    }
}

File: HIMS/wwwroot/HIMS_FrontEnd/src/app/DTOs/CreateAppointmentDto.ts
export interface CreateAppointmentDto {
    firstName: string;
    middleName?: string;
    lastName: string;
    contactNumber: string;
    
    bloodGroup: string;
    gender: string;
    address: string;
    email: string;
    dateOfBirth: string;
    
    doctorId: string;
    appointmentDate: string; 
    reasonForVisit: string;
    createdBy: string;
}